CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added `VideoPriorityBasedPolicyConfig` to control video downlink policy with network event response and recovery delays. Check [User Guide for Priority-based Downlink Policy](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html#user-guide-for-priority-based-downlink-policy) for more information.
+- Amazon Chime SDK Project Board Overview and Guide
```

Comment:
Should we link either to the README or the guide so that this is more easily discovered by builders? Also, to be consistent in the CHANGELOG, can you end this log with a "dot"?